src/com/jalasoft/search_files/search/search.py

```diff
@@ -0,0 +1,180 @@
+                    else:
+                        if common_name == item.get_file_name():
+                            result.append(item)
+            elif which_search == 1:
+                if isinstance(item, Directory):
+                    if common_name == item.get_dir_name():
+                        result.append(item)
+
+        return result
+
+    def search_by_exact_common_extension(self, path_absolute, common_extension):
+        """
+                Args:
+                path_absolute (str): The first parameter.
+                common_extension (str): The second parameter.
+
+                Return:
+                Array [Asset]: The return directories and/or files list that have exact common extension
+
+                """
+        result= []
+        factory_asset = FactoryAsset()
+        factory_asset.create_asset(path_absolute)
+        file_name, separator, extension = common_extension.partition(".")
+        for item in factory_asset.get_list_actual_directories_and_files():
+            if isinstance(item, File):
+                if extension == item.get_extension():
+                    result.append(item)
+
+        return result
+
+    def search_files_by_less_than_a_size(self, path_absolute, size_on_megabites):
+        """
+                Args:
+                path_absolute (str): The first parameter.
+                size_on_megabites (int): The second parameter.
+
+                Return:
+                Array [Asset]: The return directories and/or files list that have less than a size
+
+                """
+        result= []
+        factory_asset = FactoryAsset()
+        factory_asset.create_asset(path_absolute)
+        for item in factory_asset.get_list_actual_directories_and_files():
+            if isinstance(item, File):
+                if  item.get_size()/(1024*1224) < size_on_megabites:
+                    result.append(item)
+
+        return result
+
+    def search_files_by_bigger_than_a_size(self, path_absolute, size_on_megabites):
+        """
+                Args:
+                path_absolute (str): The first parameter.
+                size_on_megabites (int): The second parameter.
+
+                Return:
+                Array [Asset]: The return directories and/or files list that have bigger than a size
+
+                """
+        result= []
+        factory_asset = FactoryAsset()
+        factory_asset.create_asset(path_absolute)
+        for item in factory_asset.get_list_actual_directories_and_files():
+            if isinstance(item, File):
+
+                if  item.get_size()/(1024*1224) > size_on_megabites:
+                    result.append(item)
+                    print("item.get_size()/(1024*1224)", item.get_size() / (1024 * 1224))
+
+
+        return result
+
+
+    def print_list_all(self, list_to_be_print):
+        for item in list_to_be_print:
+            print(item.get_path() , "test sixe" ,item.get_size()/(1024*1224) )
```

Comment:
please change the position of the comma, just like below:
print(item.get_path() , "test sixe", item.get_size()/(1024*1224) )